src/collect/cli.rs

```diff
@@ -95,6 +95,18 @@ Examples of meta filters:
 --filter-meta 'sk_buff.dev.nd_net.net.ns.inum == 4026531840'"#
     )]
     pub(super) meta_filter: Option<String>,
+    #[arg(
+        short,
+        long,
+        default_value = "false",
+        help = "Auto-probe mode evaluates where Retis could add additional probes based on functions reported
```

Comment:
Needless to say this is a matter of personal taste (like always happens with names).
I don't find `auto-probe` very descriptive (in terms of this particular feature).
Have you considered something more explicit like `--follow-stack-trace`?
I can understand that the `short` option may "suffer" about this, but I guess it's fine (especially now that we have completion).

Reply:
Or maybe `--probe-stack`? Having a long option only is fine I guess for this one.

Reply:
sounds better to me